SDPi_Supplement/asciidoc/volume0/tf0-ch-d-glossary.adoc

```diff
@@ -90,6 +90,13 @@
 | https://profiles.ihe.net/GeneralIntro/ch-D.html[IHE TF-0 Appendix D Glossary]
 | IHE
 
+| [[term_fast_healthcare_interoperability_resources,Fast Healthcare Interoperability Resources (FHIR)]] Fast Healthcare Interoperability Resources
+| An <<acronym_hl7>> standard for health care data exchange, built on RESTful technology that utilizes "resources" to enable rapid creation of interoperable healthcare applications.
```

Comment:
Good question ... I think it was because not everyone who reads this will have a clue what RESTful is or "resources" .. perhaps I should italicize instead.  I didn't want people to confuse the specialized use of this term with the general English word "resource".

Reply:
Made the word italicized.